.pipelines/pipeline.yaml

```diff
@@ -380,28 +386,10 @@ stages:
       os: windows
       clusterType: windows-cniv1-up
       clusterName: 'win22e2e'
+      vmSize: Standard_D4s_v3
```

Comment:
Did we try a lower VM than this ? The price is way more than the default ones we have.
Since this will be a part of every PR. The cost will increase a lot for our team

Reply:
Yes, using anything lower results in NPM e2e test failure. Per @huntergregory, HNS could run out of memory.

Reply:
Then we run only on windows ? and not on linux.

Reply:
After testing, I confirmed that Linux clusters can use Standard_B2 and Windows clusters can use Standard_B2ms. These are the bare minimum and increasing the node count for either will over commit the VMs.

Reasoning for the larger vm on windows is due to HNS running on windows clusters. Decreasing it to a Standard_B2 will over commit the VM as it only has 4 GB of RAM versus the 8 GB of B2ms. Below shows the usage with Standard_B2ms VM on 2 nodes.
**Linux System pool**
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.)
  Resource               Requests        Limits
  cpu                        330m (17%)  1500m (78%)
  memory                240Mi (4%)    2786Mi (52%)
  ephemeral-storage  0 (0%)         0 (0%)
  hugepages-1Gi      0 (0%)           0 (0%)
  hugepages-2Mi      0 (0%)          0 (0%)

**Windows agent pool**
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.)
  Resource               Requests        Limits
  cpu                        420m (22%)   250m (13%)
  memory                 590Mi (13%)  2112Mi (49%)
  ephemeral-storage  0 (0%)          0 (0%)

Reply:
> Then we run only on windows ? and not on linux.
That's right, the other NPM pipepline tests are designed to run on only Linux and only Windows